webpack.config.js

```diff
@@ -150,12 +162,39 @@ module.exports = {
                 ],
                 '@babel/plugin-proposal-object-rest-spread',
               ],
+              generatorOpts: {
+                compact: false,
+              },
+            },
+          },
+        ],
+      },
+      {
+        test: /.css$/i,
+        use: [
+          {
+            loader: MiniCssExtractPlugin.loader,
+          },
+          {
+            loader: 'css-loader',
+            options: {
+              importLoaders: 1,
+              url: filterCssImport,
+              import: filterCssImport,
```

Comment:
```suggestion
              import: filterCssImport,
              sourceMap: true,
```
Otherwise debugging Fomantic styling is nearly impossible